roles/splunk/vars/RedHat.yml

```diff
@@ -1,19 +1,4 @@
 ---
 global_bashrc: /etc/bashrc
 chk_config_cmd: chkconfig --add disable-thp
-linux_packages:
```

Comment:
Can you rebase this

Reply:
My bad - thought I had done my cherry pick correctly and I did not.

Reply:
Fixed in force push